core/settings.py

```diff
@@ -603,12 +617,13 @@ def print_help():
           --only-recoubs        only download recoubs during channel downloads
 
         Misc. options:
-          --audio-only          only download audio streams
-          --video-only          only download video streams
-          --write-list FILE     write all parsed coub links to FILE
-          --use-archive FILE    use FILE to keep track of already downloaded coubs
-          --print-json FILE     output basic coub infos as JSON to FILE
-                                  see --output for the currently available infos
+          --audio-only              only download audio streams
+          --video-only              only download video streams
+          --write-list FILE         write all parsed coub links to FILE
+          --unavaiable-list FILE    write unavaiable coubs links to FILE
```

Comment:
```suggestion
          --unavailable-list FILE   use FILE to keep track of unavailable coubs
```

Also please change all instance of `unavaiable` to `unavailable` please.